src/server.js

```diff
@@ -454,4 +465,4 @@ editorTools.listen(process.env.EDITOR_PORT ? process.env.EDITOR_PORT : 4000, err
   }
 
   console.log('Editor tools server started on port', process.env.EDITOR_PORT ? process.env.EDITOR_PORT : 4000); // eslint-disable-line no-console
-});
+  });
```

Comment:
Just add a quick newline at the end of the file here. Also, is there a reason that you added a space to that closing brace? I don't think that's correct, yeah?